lib/api/core/models/repositories/repository.js

```diff
@@ -114,12 +112,6 @@ class Repository {
    * @returns {Promise<object>}
    */
   loadMultiFromDatabase (ids) {
-    if (!Array.isArray(ids)) {
```

Comment:
This module is only called by internal function so we should trust that its parameters are correct.